.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,10 +10,3 @@
 
 **Special notes for your reviewer**:
 
-**Release note**:
```

Comment:
This shouldn't be in here (right?)

Also it's probably best to squash to one commit